utils/handle-sshpw

```diff
@@ -47,8 +47,11 @@ for ud in userdata:
             users.set_user_password(username=ud.username, password=ud.password,
                                     is_crypted=ud.isCrypted, lock=ud.lock)
     else:
-        users.create_user(username=ud.username, password=ud.password, is_crypted=ud.isCrypted, lock=ud.lock,
-                          homedir=ud.homedir, shell=ud.shell, gecos=ud.gecos, root="/")
+        users.create_user(
+            username=ud.username, password=ud.password, is_crypted=ud.isCrypted, lock=ud.lock,
+            homedir=getattr(ud, "homedir", None), shell=getattr(ud, "shell", None),
```

Comment:
This bug was introduced in https://github.com/rhinstaller/anaconda/commit/1278b3f612a55e0b8dda82b0ca65b0afc46fceba. The `sshpw` command doesn't have options for `homedir`, `shell` or `gecos` (see https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#sshpw), so you can remove the optional arguments instead of using `getattr`.

Reply:
Right, that's better...